pkg/sdk/testint/tables_integration_test.go

```diff
@@ -1034,14 +1052,39 @@ func TestInt_TablesShowByID(t *testing.T) {
 		require.Equal(t, id2, e2.ID())
 	})
 
-	t.Run("show by id: check fields", func(t *testing.T) {
+	t.Run("show by id: check schema evolution record", func(t *testing.T) {
 		name := random.AlphaN(4)
-		id1 := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, name)
+		id := sdk.NewSchemaObjectIdentifier(databaseTest.Name, schemaTest.Name, name)
 
-		createTableHandle(t, id1)
+		columns := []sdk.TableColumnRequest{
+			*sdk.NewTableColumnRequest("c1", sdk.DataTypeNumber).WithDefaultValue(sdk.NewColumnDefaultValueRequest().WithIdentity(sdk.NewColumnIdentityRequest(1, 1))),
+		}
+		err := client.Tables.Create(ctx, sdk.NewCreateTableRequest(id, columns).WithEnableSchemaEvolution(sdk.Pointer(true)))
+		require.NoError(t, err)
+		t.Cleanup(cleanupTableHandle(id))
+
+		table, err := client.Tables.ShowByID(ctx, id)
+		require.NoError(t, err)
 
-		sl, err := client.Tables.ShowByID(ctx, id1)
+		err = client.Grants.GrantPrivilegesToAccountRole(ctx,
+			&sdk.AccountRoleGrantPrivileges{SchemaObjectPrivileges: []sdk.SchemaObjectPrivilege{sdk.SchemaObjectPrivilegeEvolveSchema}},
+			&sdk.AccountRoleGrantOn{SchemaObject: &sdk.GrantOnSchemaObject{SchemaObject: &sdk.Object{ObjectType: sdk.ObjectTypeTable, Name: sdk.NewObjectIdentifierFromFullyQualifiedName(table.ID().FullyQualifiedName())}}},
+			sdk.NewAccountObjectIdentifier("ACCOUNTADMIN"),
+			nil)
 		require.NoError(t, err)
-		assert.Equal(t, "ROLE", sl.OwnerRoleType)
+
+		stage, stageCleanup := testClientHelper().Stage.CreateStageInSchema(t, sdk.NewDatabaseObjectIdentifier(testDb(t).Name, schemaTest.Name))
+		t.Cleanup(stageCleanup)
+		filename := uploadFile(t, stage)
+
+		_, err = client.QueryUnsafe(ctx, fmt.Sprintf(`COPY INTO %s
+		FROM @%s/%s
+		FILE_FORMAT = (type=json)
+		MATCH_BY_COLUMN_NAME = CASE_INSENSITIVE`, table.ID().FullyQualifiedName(), stage.ID().FullyQualifiedName(), filename))
+		require.NoError(t, err)
+
+		currentColumns := getTableColumnsFor(t, client, table.ID())
+		require.Len(t, currentColumns, 2)
+		assert.NotEmpty(t, currentColumns[1].SchemaEvolutionRecord)
```

Comment:
is the result changing between the invocations or can we expect a specific value here?